src/sentry/grouping/ingest/hashing.py

```diff
@@ -85,7 +83,6 @@ def _calculate_event_grouping(
                 event.data["grouping_config"] = get_grouping_config_dict_for_project(project)
                 hashes = event.get_hashes()
 
-        hashes.write_to_event(event.data)
```

Comment:
`get_hashes()` now takes care of writing to the event:
<img width="635" alt="image" src="https://github.com/user-attachments/assets/584af685-9868-4fd7-a4c1-cbf7f48af662">